api/src/certification/configuration/routes.js

```diff
@@ -1,5 +1,7 @@
 import * as complementaryCertification from './application/complementary-certification-route.js';
+import * as scoWhitelist from './application/sco-whitelist-route.js';
 
 const certificationConfigurationRoutes = [complementaryCertification];
+const scoWhitelistRoutes = [scoWhitelist];
```

Comment:
Elle devrait pas être là plutôt? 
```suggestion
const certificationConfigurationRoutes = [complementaryCertification, scoWhitelist];
```